csharp-intermediate-topics/RenameFilesWithCSharp/RenameFilesWithCSharp/FindingFilesInFolder.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace RenameFilesWithCSharp;
+
+public class FindingFilesInFolder
+{
+    public static List<string> FindFilesInFolder(string directoryPath)
+    {
+
+        var files = Directory.GetFiles(directoryPath, "*", SearchOption.AllDirectories);
```

Comment:
We should use EnumerateFiles since we are going to just turn this into a list. GetFiles returns a string[]